modules/ROOT/pages/sap/sap-s4-hana-connector-reference.adoc

```diff
@@ -0,0 +1,518 @@
+:toc:               left
+:toc-title:         SAP S 4HANA Module
```

Comment:
Double checked the SAP website: The name is  SAP S/4HANA.  Please change all occurrences to this.